_posts/2025-03-13-CPANSec-Retrospective-2024.md

```diff
@@ -0,0 +1,146 @@
+- [CVE-2024-23525: Spreadsheet::ParseXLSX XML external entity attack vulnerability](https://security.metacpan.org/2024/02/10/vulnerable-spreadsheet-parsing-modules.html#cve-2024-23525-spreadsheetparsexlsx-xml-external-entity-attack-vulnerability)
+
+### Insecure modes of CPAN installers
+CPANSec continued with CPAN installer [cpanminus]() (a major installer of the Perl ecosystem)
+- [CVE-2024-45321: App::cpanminus through 1.7047 downloads code using insecure HTTP](https://security.metacpan.org/2024/08/26/cpanminus-downloads-code-using-insecure-http.html#cve-2024-45321)
+
+This latest is a bit related to a fix from a CPANSec member earlier in 2023 to the CPAN installer [cpan]() (part of Perl core distribution)
+- [CVE-2023-31484: CPAN.pm before 2.35 does not verify TLS certificates when downloading distributions over HTTPS](https://nvd.nist.gov/vuln/detail/CVE-2023-31484)
+
+### Other
+Similarly, a CVE was emitted for the [POSIX::2008](https://metacpan.org/dist/POSIX-2008) module:
+- [CVE-2024-55564: POSIX::2008 package before 0.24 for Perl has a potential execve50c env buffer overflow](https://nvd.nist.gov/vuln/detail/CVE-2024-55564)
+
+## Improving CVE tooling
+Along with CVEs, tooling to monitor, fetch or test for them is very useful.
+They can be installed in workflows or tooling as a gate keeper 
+for companies to do not ship vulnerable piece of code. 
+
+CPANSec produced and improved a few tools to fetch or test for CVEs:
+- [CPANSEC::Admin::Command::CVEScan](https://metacpan.org/pod/CPANSEC::Admin::Command::CVEScan)
+- [Net::NVD](https://metacpan.org/pod/Net::NVD)
+- [Net::CVE](https://metacpan.org/pod/Net::CVE)
+- [Test::CVE](https://metacpan.org/pod/Test::CVE)
+
+Note: Not all CVE tooling are managed by CPANSec, I'm thinking in particular [CPAN::Audit](https://metacpan.org/pod/CPAN::Audit) and its sidecar [CPAN::Audit::DB](https://metacpan.org/pod/CPAN::Audit::DB) or [NIST::NVD](https://metacpan.org/pod/NIST::NVD).
+But CPANSec follow closely and contribute to them when possible.
+
+## CPANSec as a CVE Numbering Authority (CNA)
+In 2023 and 2024, CPANSec helped with or contributed CVEs for Perl, 
+but assignation remained under control of another CVE Numbering Authority (CNA).
+
+CPANSec worked through the year to get more empowerment and ability to assign CVEs.
+After all, who knows better than CPANSec the impact of CVEs on Perl modules :)
+
+And finally, by end of 2024, CPANSec has laid the groundwork to become a CNA.
+
+This designation was officially granted in February 2025: [CPANSec is now a CVE Numbering Authority (CNA)!](https://security.metacpan.org/2025/02/25/cpansec-is-cna-for-perl-and-cpan.html)
+
+The news of CPANSec becoming CVE Numbering Authority (CNA) was saluted by big names like Greg Kroah-Hartman (GNU/Linux) and Daniel Stenberg (curl).
```

Comment:
s/big names/community luminaries/ ? :grin: 

Also, if their salutations were in public, let's link to them :-)